includes/components/class-attribute.php

```diff
@@ -1410,6 +1418,29 @@ public function set_range_values( $form_args, $form ) {
 		return $form_args;
 	}
 
+	/**
+	 * Adds registration fields.
+	 *
+	 * @param array  $form_args Form arguments.
+	 * @param object $form Form object.
+	 * @return array
+	 */
+	public function add_registration_fields( $form_args, $form ) {
+
+		// Get model.
+		$model = $form::get_meta( 'model' );
```

Comment:
It's ok to use hard-coded model name "user" since the hook above is user-specific, this model is the only exception (other models are post-based so we can't be sure)